src/olympia/blocklist/cron.py

```diff
@@ -20,12 +22,22 @@ def upload_mlbf_to_kinto():
     server = KintoServer(KINTO_BUCKET, KINTO_COLLECTION_MLBF)
     stats = {}
     key_format = get_mlbf_key_format()
+    signing_datetime = int(time.time() * 1000)
```

Comment:
You need to explain to an old man what this actually doing, I'm not sure I've seen time.time() being multiplied by `1000` before :thinking:

Reply:
heh, it's just to get time in milliseconds, but as a integer.  Kinto uses the same timestamp format for the record `modified` property.